src/recipes.jl

```diff
@@ -192,6 +193,7 @@ Waypoints along edges:
         elabels_fontsize = labels_theme.fontsize,
         elabels_attr = (;),
         # self edge attributes
+        force_straight_edges = true,
```

Comment:
Is this breaking?

Reply:
whoops, should be false per default! I first called it `allow_curved_edges`,  I changed the name but not the default...